web/src/app/main/components/NavSubMenu.js

```diff
@@ -0,0 +1,113 @@
+import React from 'react'
+import { NavLink } from 'react-router-dom'
+import List from '@material-ui/core/List'
+import { makeStyles } from '@material-ui/core/styles'
+import ListItem from '@material-ui/core/ListItem'
+import ListItemText from '@material-ui/core/ListItemText'
+import ListItemIcon from '@material-ui/core/ListItemIcon'
+import Typography from '@material-ui/core/Typography'
+import { styles } from '../../styles/materialStyles'
+import ExpandMoreIcon from '@material-ui/icons/ExpandMore'
+import { Collapse } from '@material-ui/core'
+import { urlPathSelector } from '../../selectors/url'
+import { useSelector } from 'react-redux'
+import { PropTypes as p } from 'prop-types'
+
+const useStyles = makeStyles(theme => {
+  const { nav, navSelected } = styles(theme)
+  return {
+    nav,
+    navSelected,
+    subMenu: {
+      padding: '0',
+    },
+    parentItem: {
+      color: theme.palette.primary['500'],
+    },
+    subMenuLinkText: {
+      paddingLeft: '3.5rem',
+      '& span': {
+        fontSize: '.9rem',
+      },
+    },
+    dropdown: {
+      transition: theme.transitions.create(['transform'], {
+        duration: theme.transitions.duration.short,
+      }),
+    },
+    dropdownOpen: {
+      transform: 'rotate(0)',
+    },
+    dropdownClosed: {
+      transform: 'rotate(-90deg)',
+    },
+  }
+})
+
+export default function NavSubMenu(props) {
+  const { parentIcon, parentTitle, path, subMenuRoutes } = props
+  const classes = useStyles()
+  const pathname = useSelector(urlPathSelector)
+  let isRoute = pathname.startsWith(path)
+
+  function renderParentLink(IconComponent, label) {
+    return (
+      <ListItem button tabIndex={-1}>
+        <ListItemIcon>
+          <IconComponent />
+        </ListItemIcon>
+        <ListItemText
+          className={classes.parentItem}
+          disableTypography
+          primary={<Typography variant='subtitle1'>{label}</Typography>}
+        />
+        <ExpandMoreIcon
+          className={
+            (classes.dropdown,
```

Comment:
We need to join these together in a string -- as committed it will only add the open/closed class and omit the `dropdown` class thus no animation.

I'm assuming it was meant to be:
```suggestion
            classnames(classes.dropdown,
```

Don't forget to add `import classnames from 'classnames'`